services/storage-users/pkg/config/config.go

```diff
@@ -138,6 +138,7 @@ type OCISDriver struct {
 	MaxConcurrency          int    `yaml:"max_concurrency" env:"STORAGE_USERS_OCIS_MAX_CONCURRENCY" desc:"Maximum number of concurrent go-routines. Higher values can potentially get work done faster but will also cause more load on the system. Values of 0 or below will be ignored and the default value will be used." introductionVersion:"pre5.0"`
 	AsyncUploads            bool   `yaml:"async_uploads" env:"OCIS_ASYNC_UPLOADS" desc:"Enable asynchronous file uploads." introductionVersion:"pre5.0"`
 	MaxQuota                uint64 `yaml:"max_quota" env:"OCIS_SPACES_MAX_QUOTA;STORAGE_USERS_OCIS_MAX_QUOTA" desc:"Set a global max quota for spaces in bytes. A value of 0 equals unlimited. If not using the global OCIS_SPACES_MAX_QUOTA, you must define the FRONTEND_MAX_QUOTA in the frontend service." introductionVersion:"pre5.0"`
+	DisableVersioning       bool   `yaml:"disable_versioning" env:"STORAGE_USERS_OCIS_DISABLE_VERSIONING" desc:"Disables versioning for files. When set to true new uploads will overwrite existing files instead of creating a new version." introductionVersion:"6.0"`
```

Comment:
```suggestion
	DisableVersioning       bool   `yaml:"disable_versioning" env:"STORAGE_USERS_OCIS_DISABLE_VERSIONING" desc:"Disables versioning of files. When set to true, new uploads with the same filename will overwrite existing files instead of creating a new version." introductionVersion:"6.0"`
```
Why not adding a global envvar because of `STORAGE_SYSTEM_OCIS_DISABLE_VERSIONING`?